packages/ts/react-crud/src/autoform.tsx

```diff
@@ -219,12 +220,14 @@ export function AutoForm<M extends AbstractModel>({
         // Handled automatically
         return;
       }
-      const genericError = 'Something went wrong, please check all your values';
-      if (onSubmitError) {
-        onSubmitError({ error });
-      } else {
-        setFormError(genericError);
+      if (error instanceof EndpointError) {
+        if (onSubmitError) {
+          onSubmitError({ error });
+        } else {
+          setFormError(error.message);
+        }
       }
+      throw error;
```

Comment:
Having this `throw` statement that re-throws anyway, makes me think of:
- What should the user be doing to catch it? Suppose that the error is type of `EndpointError` which is feed to the `onSubmitError` callback if exists. If the `onSubmitError` exists, the user still gets an uncaught error in the browser.
- What happens in the AutoCrud? 

Or maybe I'm missing a point?